source/_integrations/home_connect.markdown

```diff
@@ -34,7 +34,7 @@ Note that it depends on the appliance and on API permissions which of the featur
 
 - Application ID: Home Assistant (or whatever name makes sense to you)
 - OAuth Flow: Authorization Code Grant Flow
-- Redirect URI  : `<YOUR_INTERNAL_HOME_ASSISTANT_URL>/auth/external/callback` # Use same the address defined as `internal_url` in your configuration settings. Example `http://192.168.1.24:8123/auth/external/callback`
+- Redirect URI  : `<YOUR_LOCAL_HOME_ASSISTANT_URL>/auth/external/callback` # Use your local IP address or hostname. Example `http://homeassistant.local:8123/auth/external/callback`
```

Comment:
How about this?

```suggestion
- Redirect URI: `<INTERNAL_HOME_ASSISTANT_URL>/auth/external/callback`
  Use your internal Home Assistant URL, if you didn't configure one manually, use your local IP address. Examples: `http://192.168.0.2:8123/auth/external/callback`, `http://homeassistant.local:8123/auth/external/callback`.
```

Reply:
Sure. That should clarify most of the configuration scenarios. :) 
The important thing is for our users to not struggle with the component trying to get it going.